### PR TITLE
Cleanup markup

### PR DESCRIPTION
**Pull request by seemantk**
Clean up the html doc, separating out all js into independent files, and the style into a stylesheet file.  Move the display attributes in notesbook to a display object.

### CODE REVIEW

**Comment by seemantk** [comment]
This PR also has a placeholder .extreme styling in the css for extreme notes

File: js/notescanvas.js [hunk 1]
```diff
@@ -86,22 +83,11 @@ function NotesCanvas() {
...
-                }
-              })
-            .attr("stroke-width", function(d) {
-                if(showExtremeNotes && isExtremeNote.get(this)){
-                    return extremeNoteStrokeWidth;
-                } else {
-                    return defaultStrokeWidth;
-                }
+            .classed("extreme", function(d) {
+                return showExtremeNotes && isExtremeNote.get(this);
```
Comment (curran) [inline js/notescanvas.js]: Nice move!

File: js/notescanvas.js [hunk 2]
```diff
@@ -96,19 +96,15 @@ function NotesCanvas() {
...
-                );
-                d3.select(this)
-                    .classed("extreme", function(d) {
-                        return showExtremeNotes && isExtremeNote.get(this);
-                      })
-                ;
-              })
+        svg.selectAll("rect.note").each(function(d) {
+            d3.select(this)
+                .classed("extreme", function(d) {
```
Comment (curran) [inline js/notescanvas.js:101]: Awesome, more direct approach.

**Comment by curran** [comment]
Thanks @seemantk, looks great!

**Comment by curran** [comment]
I noticed that the controls are now located below the visualizations. I assume this is intentional, and paves the way for filling the complete horizontal space with the SVGs for #33.

**Pull request merged**